.github/workflows/playwright.yml

```diff
@@ -103,7 +103,7 @@ jobs:
           npx wp-env run cli wp --info
       - name: Activate performance experiments
         run: |
-          npx wp-env run cli wp elementor experiments activate e_font_icon_svg,e_lazyload,e_dom_optimization,e_optimized_assets_loading,e_optimized_css_loading,additional_custom_breakpoints,rating
```

Comment:
I believe we want to keep the Playwright tag.

Reply:
@hein-obox it's about activating experiments, since we decide to delete experiment - it's not actual anymore

Reply:
@mykytamurzin I believe we use this tag for all Rating widget tests. Creating separate tags makes it easier to manage Playwright runs, and e.g. helps with running tests parallel from each other.